rootfs/Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV LANG en_US.utf8
 RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
 
 ENV PG_MAJOR 9.4
-ENV PG_VERSION 9.4.6-1.pgdg80+1
+ENV PG_VERSION 9.4.7-1.pgdg80+1
```

Comment:
I had added it so i can test.I want to merge this PR.When are you planning to merge #78 PR.Also the PR doesn't work on it's own.It failed for me during the restore.i had to add https://github.com/kmala/postgres/blob/storage/rootfs/docker-entrypoint-initdb.d/003_restore_from_backup.sh#L47-L48 to make it work.